thirdparty/cmake_modules/CMakeCross.cmake

```diff
@@ -1,9 +1,12 @@
 # CMake Cross ToolChain config file. Adapted from Debian's dpkg-cross ;).
 # c.f., https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-toolchain
+# NOTE: Remember that CMake is a special snowflake that completely *ignores* CPPFLAGS,
+# because of course it does...
+# (https://gitlab.kitware.com/cmake/cmake/issues/12928)
```

Comment:
I'm surprised that issue was only opened 8 years ago.

Reply:
:D

Reply:
I'm surprise no-one decided this was grounds to throw it into a fire, while autotools actually shouts at you for using CPPFLAGS wrong.